### PR TITLE
Feature/oe 149 customer index date sold

### PR DESCRIPTION
**Pull request by Ron-Junior**
## Goal

Add data of sale to the list of customers 

## Changeset

- next to kW on the list of the customer was add the date of sale

## Review

For the submitter, initial self-review:

- [ ] Reviewed the test cases added for completeness and possible points for discussion
- [ ] Check the scope of the changeset - is everything in the diff required for the pull request?

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency between the changeset and the goal stated above
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Complexity - is the proposed change introducing any unjustified complexity? 
- [ ] Thoroughness of added tests and any missing edge cases

### CODE REVIEW

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -70,7 +70,7 @@ class="@if($customer->is_active && $customer->panel_sold) bg-green-base @elseif(
                                         </div>
                                     </div>
                                     <div class="col-span-7 text-xs text-gray-600">
-                                        {{ number_format($customer->epc) }}kW
+                                        {{ number_format($customer->epc) }}kW - <i> {{ date('d/m/Y', strtotime($customer->date_of_sale))}} </i>
```
Comment (iammayron) [inline resources/views/home.blade.php:73]: Please, use the Carbon class to format this date. You also could cast this field on the Customer model file to date and use `$customer->date_of_sale->format('d/m/Y')`. When you cast a model field it will assume the type that you set, when you set it to date, the field will already be changed use the Carbon class.

I don't know if it is a request or a project pattern but the default date format in the USA is m/d/Y.

**Review by iammayron — APPROVED** [review]

File: resources/views/home.blade.php (hunk 1 above)
Comment (Ron-Junior) [reply resources/views/home.blade.php:73]: about of date format, if it's d/m/Y or m/d/Y... I'm waiting for the confirmation of Lucas
